packages/app/src/AppNav.tsx

```diff
@@ -854,7 +854,7 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
           }}
         >
           <AppNavHelpMenu
-            version={version}
+            version={packageJson.version}
```

Comment:
fixes a new next warning about stricter imports when importing from a `default` export

Reply:
does this still treeshake the rest of the package.json out? Or at least I recall our current bundle treeshakes the rest of the JSON out so we only actually bundle the version string